swarmlib/__main__.py

```diff
@@ -58,6 +58,11 @@ def run_swarm():
         default=None,
         type=int,
         help='Used to set the initial state of the random bit generator (default None)')
+    parser.add_argument(
+        '--log-level',
+        default='info',
+        choices=['debug', 'info', 'warning', 'error', 'critical'],
+        help='Set the log level')
```

Comment:
Just one last single bit: can you add the default in the help description, too? Similar to like it is done with the `--seed` option above. Then we are good to go

Reply:
Done :)